lib/messages/deployment-status.js

```diff
@@ -10,24 +10,30 @@ class DeploymentStatus extends Message {
         htmlURL: deployment.creator.html_url,
       },
       footer: `<${repository.html_url}|${repository.full_name}>`,
-
     });
     this.deploymentStatus = deploymentStatus;
     this.deployment = deployment;
     this.repository = repository;
-    this.shortSha = this.deployment.sha.substr(0, 7);
     this.commitLink = `${this.repository.html_url}/commit/${this.deployment.sha}`;
   }
 
+  prettyRef() {
+    return this.deployment.ref && this.deployment.ref !== this.deployment.sha
+      ? this.deployment.ref
```

Comment:
Does it make sense to include the sha alongside the ref if we do have one? So something like:

```suggestion
      ? `${this.deployment.ref} (${this.deployment.sha.substr(0, 7);})`
```

So that in Slack it shows up as 
"login is deploying `master` (abc1def2) to staging"

Reply:
That's a great idea 👏 
I Added it, added screenshot and implemented your other suggestions too.